package.json

```diff
@@ -49,6 +49,7 @@
     "util:prep-next-from-existing-build": "ts-node ./support/prepareVersionUpdate.ts --next --standard-version-options=\"--skip.changelog\"",
     "util:publish-next": "npm publish --tag next",
     "util:push-tags": "git push --follow-tags",
+    "util:build-custom-elements-output-utils": "ts-node support/output-targets/custom-elements/createWrappers.ts && tsc --project ./support/output-targets/custom-elements/tsconfig-dist-utils.json && rimraf ./extras/output-targets/custom-elements/wrappers",
```

Comment:
may be worth breaking this into multiple commands or just multiple lines? Very long.

Reply:
Agreed, some of these are getting unwieldy. 

I'll tackle this in a follow-up PR. https://www.npmjs.com/package/script-launcher could help break some of these into multiple lines, which would improve readability.